README.md

```diff
@@ -37,6 +37,13 @@ Provide the SSH key for access to Dokku in the `SSH_PRIVATE_KEY` secret.
 
 `Procfile` contains the command for running the main process of the application.
 
+## Environment variables
+
+|  Variable           | Default                    |   Meaning                                         |
+|---------------------|----------------------------|---------------------------------------------------|
+| MAX_ATTEMPTS        |    4                       | Number of restart attempts for a specific job.    |
+| STABLE_BRANCHES_LIST| {'master', '2.11', '2.10'} | Branches that require automatic restart of tasks. |
```

Comment:
I think we should leave only `master` here. Stable branches change quite often (say, in a half year 2.11 and 2.12 will be stable).

Reply:
Fixed it. Thanks!